<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="DESKTOP-2O6RHPG" name="pagesTestCasesYasasiiWeb.LoginTestcasesYW" tests="5" failures="0" timestamp="2021-09-25T23:06:29 IST" time="17.796" errors="1">
  <testcase name="userlogin" time="0.000" classname="pagesTestCasesYasasiiWeb.LoginTestcasesYW">
    <skipped/>
  </testcase> <!-- userlogin -->
  <system-out/>
  <testcase name="userlogin" time="0.000" classname="pagesTestCasesYasasiiWeb.LoginTestcasesYW">
    <skipped/>
  </testcase> <!-- userlogin -->
  <system-out/>
  <testcase name="userlogin" time="17.792" classname="pagesTestCasesYasasiiWeb.LoginTestcasesYW"/>
  <system-out/>
  <testcase name="getResult" time="0.004" classname="pagesTestCasesYasasiiWeb.LoginTestcasesYW">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.baseYasasiiWeb.TestBaseYasasiiWeb.getResult(TestBaseYasasiiWeb.java:202)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- getResult -->
  <system-out/>
  <testcase name="userlogin" time="0.000" classname="pagesTestCasesYasasiiWeb.LoginTestcasesYW">
    <skipped/>
  </testcase> <!-- userlogin -->
  <system-out/>
</testsuite> <!-- pagesTestCasesYasasiiWeb.LoginTestcasesYW -->
